lib/auth/auth_with_roles.go

```diff
@@ -1352,6 +1352,16 @@ func (a *ServerWithRoles) ListUnifiedResources(ctx context.Context, req *proto.L
 			actionVerbs = []string{types.VerbList}
 		}
 
+		// IdentityCenter resources are bundled under the umbrella KindIdentityCenter
+		// so we need to check that as well as the actual concrete kind
+		if kind == types.KindIdentityCenterAccount {
+			err := a.action(apidefaults.Namespace, types.KindIdentityCenter, actionVerbs...)
+			if err != nil {
+				resourceAccess.kindAccessMap[kind] = err
+				continue
+			}
```

Comment:
Nit: Isn't this the equivalent?
```suggestion
			resourceAccess.kindAccessMap[kind] = a.action(apidefaults.Namespace, types.KindIdentityCenter, actionVerbs...)
```